### PR TITLE
Address #755 to help users discover other language examples

### PR DESCRIPTION
**Pull request by ybyzek**
### Description

#755 

### Staging Docs

http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-755
http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-755/produce-consume-lang/scala.html

### CODE REVIEW

**Comment by bbejeck** [comment]
Quick question we have [Your first producer application](http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-755/creating-first-apache-kafka-producer-application/kafka.html
) and [Your first Kafka consumer application](http://kafka-tutorials-staging.s3-website-us-west-2.amazonaws.com/GH-755/creating-first-apache-kafka-consumer-application/kafka.html
) could those cover the `Java` tab?

**Comment by ybyzek** [comment]
@bbejeck great point.  Would it be preferred if I reframed the other use case as non-Java only (in kind of was originally).

File: _layouts/tutorial.html [hunk 1]
```diff
@@ -69,6 +69,36 @@ <h3 class="title">Code example:</h3>
           Scala
           </option>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.java == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/java.html" {% if page.stack == 'java' %} selected {% endif %}>
+          Java
+          </option>
+          {% endif %}
```
Comment (bbejeck) [inline _layouts/tutorial.html:76]: with your latest changes, we could probably get rid of this?

File: _layouts/tutorial.html [hunk 2]
```diff
@@ -93,6 +123,36 @@ <h3 class="title">Code example:</h3>
             Scala
           </a>
           {% endif %}
+          {% if site.data.tutorials[page.static_data].status.java == 'enabled' %}
+          <a class="button {% if page.stack == 'java' %} is-selected {% endif %}" href="{{ site.data.tutorials[page.static_data].slug | relative_url }}/java.html">
+            Java
+          </a>
+          {% endif %}
```
Comment (bbejeck) [inline _layouts/tutorial.html:130]: same here

File: tutorials/produce-consume-lang/java.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+layout: tutorial
```
Comment (bbejeck) [inline tutorials/produce-consume-lang/java.html:2]: same here

**Review by bbejeck — COMMENTED** [review]
Overall looks good to me, I just have a couple of minor comments.

File: _layouts/tutorial.html (hunk 1 above)
Comment (ybyzek) [reply _layouts/tutorial.html:76]: Good catch, thanks.  Addressing...

File: _layouts/tutorial.html (hunk 2 above)
Comment (ybyzek) [reply _layouts/tutorial.html:130]: Good catch, thanks.  Addressing...

File: tutorials/produce-consume-lang/java.html (hunk 3 above)
Comment (ybyzek) [reply tutorials/produce-consume-lang/java.html:2]: Good catch, thanks.  Addressing...

File: _layouts/tutorial.html [hunk 4]
```diff
@@ -69,6 +69,31 @@ <h3 class="title">Code example:</h3>
...
+          {% if site.data.tutorials[page.static_data].status.go == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/go.html" {% if page.stack == 'go' %} selected {% endif %}>
+          Go
+          </option>
+          {% endif %}
+          {% if site.data.tutorials[page.static_data].status.c == 'enabled' %}
+          <option value="{{ site.data.tutorials[page.static_data].slug | relative_url }}/c.html" {% if page.stack == 'c' %} selected {% endif %}>
+          C
+          </option>
+          {% endif %}
```
Comment (bbejeck) [inline _layouts/tutorial.html:91]: A minor point, but I don't see `C` showing up on the page.  But with [librdkafka](https://github.com/edenhill/librdkafka) I'm wondering if we want this option too.  But I'll leave it up to you

**Review by bbejeck — APPROVED** [review]
LGTM!

File: _layouts/tutorial.html (hunk 4 above)
Comment (ybyzek) [reply _layouts/tutorial.html:91]: Great idea @bbejeck .  Added `C (librdkafka)` -- consistent with https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/c.html#client-examples-c

WDYT?

**Pull request merged**